src/ansi.rs

```diff
@@ -660,6 +667,9 @@ pub trait Handler {
     ///
     /// The output is of form `CSI > 4 ; mode m`.
     fn report_modify_other_keys(&mut self) {}
+
+    /// The VTE handler requested to write payload to the terminal.
+    fn pty_write(&mut self, _payload: String) {}
```

Comment:
VTE writing arbitrary stuff shouldn't be a thing. There's a reason why this is a trait-based interface, we should have proper functions here.